src/main/java/org/geysermc/extension/thirdpartycosmetics/Utils.java

```diff
@@ -76,28 +80,21 @@ private static BufferedImage scale(BufferedImage bufferedImage, int newWidth, in
     /**
      * Resize a BufferedImage that is expected to be a cape
      *
-     * @param image The BufferedImage to resize
+     * @param original The BufferedImage to resize
      * @return The converted BufferedImage
      */
-    public static BufferedImage resizeCape(BufferedImage image) throws IOException {
-        if (image.getWidth() > 64 || image.getHeight() > 32) {
-            // Prevent weirdly-scaled capes from being cut off
-            BufferedImage newImage = new BufferedImage(128, 64, BufferedImage.TYPE_INT_ARGB);
-            Graphics g = newImage.createGraphics();
-            g.drawImage(image, 0, 0, image.getWidth(), image.getHeight(), null);
-            g.dispose();
-            image.flush();
-            image = scale(newImage, 64, 32);
-        } else if (image.getWidth() < 64 || image.getHeight() < 32) {
-            // Bedrock doesn't like smaller-sized capes, either.
-            BufferedImage newImage = new BufferedImage(64, 32, BufferedImage.TYPE_INT_ARGB);
-            Graphics g = newImage.createGraphics();
-            g.drawImage(image, 0, 0, image.getWidth(), image.getHeight(), null);
-            g.dispose();
-            image.flush();
-            image = newImage;
-        }
-        return image;
+    public static BufferedImage resizeCape(BufferedImage original) {
+        int imageWidth = 64;
+        int imageHeight = 32;
+
+        for (int srcWidth = original.getWidth(), srcHeight = original.getHeight(); imageWidth < srcWidth || imageHeight < srcHeight; imageWidth *= 2, imageHeight *= 2) {}
```

Comment:
Can we replace this with something easier to read/understand? EG:
```suggestion
        int srcWidth = original.getWidth();
        int srcHeight = original.getHeight();
        while (imageWidth < srcWidth || imageHeight < srcHeight) {
                imageWidth *= 2;
                imageHeight *= 2;
        }
```